### PR TITLE
Rename community level to community risk level

### PR DESCRIPTION
**Pull request by faihegberg**
Rename instances of "community level(s)" to "community risk level(s)" (except explicit mentions of "CDC's community levels"), as per [card](https://trello.com/c/kIxaEm0s/647-update-all-mentions-of-cans-community-level-to-community-risk-level-leave-the-cdc-community-level-language-as-is).

### CODE REVIEW

File: scripts/alert_emails/template.html [hunk 1]
```diff
@@ -48,11 +48,13 @@
                                 style="font-size:26px;line-height:32px;Margin-top:30px;Margin-bottom:40px;color:#c3ced9;Margin-left:20px;Margin-right:20px;border-width:0!important;border-style:none!important;border-color:white!important">
                                 <div align="center" id="m_7975860115323008578emb-email-heaÞr"
                                     style="line-height:inherit;border-width:0!important;border-style:none!important;border-color:white!important">
-                                    <a href="https://covidactnow.org?utm_source=risk_alerts&utm_medium=email" style="line-height:inherit" target="_blank">
+                                    <a href="https://covidactnow.org?utm_source=risk_alerts&utm_medium=email"
+                                        style="line-height:inherit" target="_blank">
```
Comment (faihegberg) [inline scripts/alert_emails/template.html:52]: Changes to this file are much smaller than what is displayed here. This is just the linter re-organizing white space!

**Comment by faihegberg** [comment]
[Update all mentions of CAN's "Community Level" to "Community Risk Level".  Leave the "CDC Community Level" language as-is](https://trello.com/c/kIxaEm0s/647-update-all-mentions-of-cans-community-level-to-community-risk-level-leave-the-cdc-community-level-language-as-is)

**Review by chasulin — APPROVED** [review]

File: scripts/alert_emails/template.html (hunk 1 above)
Comment (chasulin) [reply scripts/alert_emails/template.html:52]: is there a setting in your linter that doesnt match CAN's?

Comment (faihegberg) [reply scripts/alert_emails/template.html:52]: It might be! This is the second time it's happening so I'll take a look 👀

**Pull request merged**